GoldenPC.yml

```diff
@@ -171,6 +171,18 @@ services:
         command: "nice -n -8 /usr/bin/opendlv-device-cangw-rhino --can=can0 --cid=101 --id=1"
 #        command: "nice -n -8 /usr/bin/opendlv-device-cangw-rhino --can=can0 --enablethrottle --enablebrake --enablesteering --cid=101 --id=1"
 
+#    opendlv-device-can-raw:
```

Comment:
This should be updated to the uint64 version that I need to merge with master and release.